package.json

```diff
@@ -8,6 +8,8 @@
     "test": "turbo run test",
     "lint": "turbo lint",
     "lint:fix": "turbo run lint:fix",
+    "generate:assets": "turbo run generate:assets && npm run generate:readme:toc",
```

Comment:
during work on this template, when I added new section in readme I noticed TOC is not updated - we forgot about enabling TOC update automation when monorepo was included - not it is done, also generator api docs refresh is also now fixed